lib/Elastica/Script/ScriptFields.php

```diff
@@ -1,7 +1,11 @@
 <?php
+
+declare(strict_types=1);
```

Comment:
please, as stated here https://github.com/ruflin/Elastica/issues/1548#issuecomment-447821664, remove the strict_types declaration

Reply:
In this case, we should remove one test because `int(0)` converts to `string("0")` silently

Reply:
Can you tell me where this happen? I can't find it :(

Reply:
[Here](https://github.com/ruflin/Elastica/blob/82c640445ec754f4b4fcd04d3a787ef43b47e360/test/Elastica/Script/ScriptFieldsTest.php#L63) is that test.

Reply:
@osavchenko can you explain me better the problem ?

Reply:
@p365labs as you can see from that test we are passing a plain array and it will have numeric indices (0, 1, 2, ...). Without typization we have check that key should be string and nut empty. With typization numeric array index will be converted silently to string ("0", "1", "2", ...) and that test will not pass.